### ISSUE TITLE
ConvertGremlinToGraphTraversalCode function changes property value through regex

### ISSUE THREAD

**Issue by dotsad** [Azure/azure-documentdb-dotnet#355]
Microsoft.Azure.Graphs.Common.TraversalScriptUtilities has a ConvertGremlinToGraphTraversalCode function that changes a value of the property through regex.

We use Azure KeyVault to encrypt an email value and store it in cosmosDB

Example `"g.addV('Person').property('Email', 'Ht3c+sgUU5yIKZ2fsj7aP7cP+eVTn9jCSOesS3jmcfLwpFzf4g2EbgzIaPjuHS7vHLPl0RCU+sKCi6DqG2eHhi76ivvbufBOkTixkC+iDGm67ImZFTZln6chAYR7OxGKesOOJfkO8SmXu7FflTaA+o0FIJ9QtedoEs4LUy5hQWXS8u3l3ZBiCBuGo8gkx4Rk32KciW3Lc+LQBvegAVapS/IK1qfdwhqcRFFEABILXoz9t9AfiDzNyL7fS/0EGGYuUxnzM5YCig3ieDtC7LiKU3BKIYLLfr7jQCKQ5XSVY8iKacCwJXUXRt8TATgcidEqFv05QqULCuz643K88ZZy8w==')";
`

As soon as the code goes through these two lines, 'l' low case will be changed to "L" upper case

```
Regex regex = new Regex("(?<!\\\"[\\s\\w]*)(\\d+)l");
text = regex.Replace(text, "$1L");
```
original value pre regex change

> Ht3c+sgUU5yIKZ2fsj7aP7cP+eVTn9jCSOesS3jmcfLwpFzf4g2EbgzIaPjuHS7vHLPl0RCU+sKCi6DqG2eHhi76ivvbufBOkTixkC+iDGm67ImZFTZln6chAYR7OxGKesOOJfkO8SmXu7FflTaA+o0FIJ9QtedoEs4LUy5hQWXS8u3 --->**l**<--- 3ZBiCBuGo8gkx4Rk32KciW3Lc+LQBvegAVapS/IK1qfdwhqcRFFEABILXoz9t9AfiDzNyL7fS/0EGGYuUxnzM5YCig3ieDtC7LiKU3BKIYLLfr7jQCKQ5XSVY8iKacCwJXUXRt8TATgcidEqFv05QqULCuz643K88ZZy8w==

changed value

> Ht3c+sgUU5yIKZ2fsj7aP7cP+eVTn9jCSOesS3jmcfLwpFzf4g2EbgzIaPjuHS7vHLPl0RCU+sKCi6DqG2eHhi76ivvbufBOkTixkC+iDGm67ImZFTZln6chAYR7OxGKesOOJfkO8SmXu7FflTaA+o0FIJ9QtedoEs4LUy5hQWXS8u3 --->**L**<---  3ZBiCBuGo8gkx4Rk32KciW3Lc+LQBvegAVapS/IK1qfdwhqcRFFEABILXoz9t9AfiDzNyL7fS/0EGGYuUxnzM5YCig3ieDtC7LiKU3BKIYLLfr7jQCKQ5XSVY8iKacCwJXUXRt8TATgcidEqFv05QqULCuz643K88ZZy8w==

As we get a value back from CosmosDb we obviously can not decrypt as value changed and `Error occurred while decoding OAEP padding` will be thrown.

Thanks.

**Comment by bartdotnet** [comment]
in my opnion this is a regression bug

**Comment by bartdotnet** [comment]
is this a regression of following issue? https://github.com/Azure/azure-documentdb-dotnet/issues/280

**Comment by LuisBosquez** [comment]
@dotsad Which version of the Microsoft.Azure.Graphs library are you using? This bug should have been resolved by version `>=0.3.0-preview`.

**Comment by rnagpal** [comment]
This issue has been fixed in the latest SDK version as mentioned above.
